.github/CODEOWNERS

```diff
@@ -4,5 +4,8 @@
 # These owners and first officers will be the default owners for everything in the repo.
 *   @BitGo/engineering
 
+/* @BitGo/ethalt-team
+/* @BitGo/btc-team
+
 ## THIS SHOULD ALWAYS BE LAST ##
-/.github/CODEOWNERS @BitGo/git-admins
+/.github/CODEOWNERS @BitGo/git-admins
```

Comment:
Please fix your editor configuration to ensure that it maintains a trailing newline at the end of the file, rather than removing it.